scrape/scrape.go

```diff
@@ -346,6 +354,7 @@ func (sp *scrapePool) stop() {
 		targetScrapePoolTargetLimit.DeleteLabelValues(sp.config.JobName)
 		targetScrapePoolTargetsAdded.DeleteLabelValues(sp.config.JobName)
 		targetSyncIntervalLength.DeleteLabelValues(sp.config.JobName)
+		targetSyncFailed.DeleteLabelValues(sp.config.JobName)
```

Comment:
Do we initiate it for each scrape job?

Reply:
I believe so

https://github.com/prometheus/prometheus/blob/212e6b91b63e163b47391cc2785e4e8355365aae/scrape/scrape.go#L461